src/docfx/Errors.cs

```diff
@@ -88,6 +88,13 @@ public static Error MicrosoftGraphApiFailed(string exMessage)
 
             public static Error MetadataValidationRuleset(string ruleset)
                 => new Error(ErrorLevel.Info, "MetadataValidationRuleset", $"Metadata validation ruleset used: {ruleset}");
+
+            /// <summary>
+            /// Liquid is not found for curtrent mime type.
+            /// </summary>
+            /// Behavior: ❌ Message: ❌
+            public static Error LiquidNotFound(SourceInfo<string?> source)
+                => new Error(ErrorLevel.Warning, "liquid-not-found", $"Liquid used to genenrate HTML is not found for mimeType '{source}'", source);
```

Comment:
`Liquid` -> `Liquid template`
and better to add `The output HTML will not be generated.`

Reply:
typo: `generate`